config/config.go

```diff
@@ -36,10 +53,12 @@ func NewViper() *viper.Viper {
 
 	v.MustBindEnv("kubectl", "KUBECTL_COMMAND")
 	v.MustBindEnv("objfreshthreshold", "KUBECOLOR_OBJ_FRESH")
+	v.MustBindEnv("pager", "KUBECOLOR_PAGER") // Don't bind PAGER yet as it should be overwritten by the config file
```

Comment:
I believe we don't need to bind the env here as we're using `v.AutomaticEnv()` above.

The other `v.MustBindEnv` here are used to bind non-automatic environment variables (where the env var name was picked before the config file feature was introduced, i.e legacy names)


```suggestion
	// NOTE: Don't bind PAGER here as it should be overwritten by the config file
```

Reply:
I also thought `AutomaticEnv()` should suffice, but in testing it didn't work. Not sure why. Do you want this to be figured out or is this okay for now?

Reply:
I did a little digging and found out that it's due to a very common bug in Viper. It was fixed in a recent version, but it's only available if you build you project with `-tags=viper_bind_struct` (https://github.com/spf13/viper/pull/1429#issuecomment-1870976604)

I'd really like to avoid requiring build flags, if possible. Especially for essential components like the configs. (would be more OK with build flags to enable niche optimizations or stuff like that, or to enable preview features)

One workaround is to set a value for the key, like via this hack:

```go
for _, key := range viper.AllKeys() {
	viper.Set(key, viper.Get(key))
}
```

But setting a default via `v.SetDefault` seems to also work. Which IMO makes the API better as we're talking about setting a default here anyways.

```go
v.SetDefault("pager", defaultPager())
```

Then you don't need this "default pager business logic" in the `./command` package, but instead it lives in the `./config` package where it should be.

This would mean that `defaultPager()` is executed even when it's not needed. But I don't believe it's an especially heavy function anyways. So should be fine.